.vortex/docs/.utils/VideoRecorder.php

```
@@ -393,6 +420,142 @@ public function applyTimeScale(string $cast_path, float $factor): void {
+    $patterns = [
+      // Operating system commands, such as the window title.
+      '#\x1b\][^\x07\x1b]*(?:\x07|\x1b\\\\)#',
+      // Control sequence introducer, covering colors and cursor movement.
+      '#\x1b\[[0-9;?]*[ -/]*[@-~]#',
+      // Two-character escapes left between the sequences above.
+      '#\x1b[@-Z\\\\-_]#',
+    ];
+
+    $stripped = preg_replace($patterns, '', $text);
+    if ($stripped === NULL) {
+      throw new RuntimeException('Failed to strip ANSI sequences from cast text');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '468', column '17'). (undefined)

(MissingImport)